### ISSUE TITLE
CI testing seems broken (yet again)

### ISSUE THREAD

**Issue by brodybits** [stryker-mutator/stryker#2067]
I seem to get an email every day that my fork has failing CI. I tried pushing updates from master to my fork then discovered that this seems to be persisting in Stryker. Here is what I have noticed so far:

- a build error that should be fixed by PR #2066
- Travis CI seems to have been broken for a while
- Karma runner not stable, sometimes but not always succeeds in my workarea, broken in CI, not sure if it is #2049 or not

Is there anything we can do to help get CI testing working again?

**Comment by simondel** [comment]
Thanks for wanting to help, I appreciate it ❤️ 

I merged this PR tonight: https://github.com/stryker-mutator/stryker/pull/2072

That should fix most issues. I missed your PR, sorry! I also changed the package.json so we no longer add versions of the dependencies that add new features because they may break our build (as we've now seen).

We're planning on moving away from Travis, but in the mean time it should be functional. Unfortunately Travis is pretty bad at sending emails. It usually only sends them to the last person that contributed :( I'll take a look at what's causing Travis to fail, unless if you want to take a look.

I believe @nicojs was working on the race condition, but I could be wrong.
